File: fizzbuzz.hs
proc :: Int -> String
proc i = case (i `mod` 3, i `mod` 5) of
    (0, 0) -> "FizzBuzz"
    (0, _) -> "Fizz"
    (_, 0) -> "Buzz"
    _ -> show i

main :: IO()
main = putStr $ unlines $ map proc [1..100]

File: fizzbuzz.r
for(i in 1:100) {
    x <- i %% 3
    y <- i %% 5
    if(x == 0) cat("Fizz")
    if(y == 0) cat("Buzz")
    if((x * y) != 0) cat(i)
    cat("\n")
}
